include/zenoh/api/session.hxx

```diff
@@ -433,14 +433,14 @@ class Session : public Owned<::z_owned_session_t> {
                                                       SubscriberOptions&& options = SubscriberOptions::create_default(),
                                                       ZResult* err = nullptr) const {
         static_assert(
-            std::is_invocable_r<void, C, const Sample&>::value,
+            std::is_invocable_r<void, C, Sample&>::value,
```

Comment:
Similar fixes need to be done for every subscriber/queryable and get function :(

Reply:
done